### PR TITLE
CLI install doc fixes

### PR DESCRIPTION
**Pull request by branden**
## Description

https://jira.mesosphere.com/browse/COPS-3411

This PR fixes up the configuration section of CLI install docs to correct or clarify guidance, makes clear which config values are required, and explains default values specific to the CLI installer. See commits for a list of individual fixes.

Tested the 1.11 Enterprise doc manually.

## Urgency
- [x] Blocker 
- [ ] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects).
- Change all affected versions (e.g. 1.7, 1.8, 1.9, 1.10, 1.11).
- See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing).

### CODE REVIEW

File: pages/1.9/installing/ent/custom/cli/index.md [hunk 1]
```diff
@@ -107,49 +107,51 @@ In this step, you create a YAML configuration file that is customized for your e
...
     
+    The CLI installer uses these default configuration values, which you may override in your configuration:
+
+    - `ssh_port`: `22`
+      - Only override this value if SSH on cluster nodes is available at a different port.
+    - `process_timeout`: `120`
+
     **Tips:** 
     
     - If Google DNS is not available in your country, you can replace the Google DNS servers `8.8.8.8` and `8.8.4.4` with your local DNS servers.
```
Comment (anithamuthu79) [inline pages/1.9/installing/ent/custom/cli/index.md]: @branden Line #120 should be removed as seen in /1.11/installing/ent/custom/cli/index.md

**Comment by branden** [comment]
@anithamuthu79 @stevenwhannah Thanks for the feedback. I pushed some fixes, PTAL.

**Comment by anithamuthu79** [comment]
@branden LGTM. Thanks for making these changes.

**Comment by stevenwhannah** [comment]
It's important to note that the customer also asked for a comprehensive BASH installation script instead of copying/pasting things into configuration files. That request for a universal installer script was not addressed in this PR.

**Pull request merged**